src/hypofuzz/database.py

```diff
@@ -287,12 +285,16 @@ def from_reports(
         elapsed_time_diff = report.elapsed_time - last_elapsed_time
         timestamp_monotonic = (
             report.timestamp
-            if last_report is None
+            if last_worker_report is None
             else max(
-                report.timestamp, last_report.timestamp_monotonic + elapsed_time_diff
+                report.timestamp,
+                last_worker_report.timestamp_monotonic + elapsed_time_diff,
             )
         )
 
+        assert elapsed_time_diff >= 0.0
+        assert timestamp_monotonic >= 0.0
```

Comment:
nope, this can happen without a bug if the user set their system clock to 1969 for some reason.  I agree that this is very strange, but it's not completely out of the question that someone would fuzz like that...

Reply:
agh, yup.